source/common/lua/wrappers.cc

```diff
@@ -0,0 +1,27 @@
+#include "common/lua/wrappers.h"
+
+namespace Envoy {
+namespace Lua {
+
+int BufferWrapper::luaByteSize(lua_State* state) {
+  lua_pushnumber(state, data_.length());
```

Comment:
Nit: might is be better to use consistent terminology for size/length across C++ and Lua?

Reply:
Sure will fix.